configure

```diff
@@ -5679,25 +5705,25 @@ define_in_config_if_yes $_nasm 'USE_NASM'
 if test "$_pandoc" = yes ; then
 	echocheck "pandoc"
 
-	IFS="${IFS= 	}"; ac_save_ifs="$IFS"; IFS=$SEPARATOR
+	test_exe_in_path pandoc
 
-	for path_dir in $_pandocpath; do
-		if test -x "$path_dir/pandoc$NATIVEEXEEXT" ; then
-			PANDOC="$path_dir/pandoc$NATIVEEXEEXT"
-			break
-		fi
-	done
+	echo $_pandoc
+fi
 
-	IFS="$ac_save_ifs"
+define_in_config_if_yes $_pandoc 'USE_PANDOC'
 
-	if ! test "$PANDOC" ; then
-		_pandoc=no
-	fi
+#
+# Check for curl
+#
+if test "$_curl" = yes ; then
+	echocheck "curl"
 
-	echo $_pandoc
+	test_exe_in_path curl
+
+	echo $_curl
 fi
 
-define_in_config_if_yes $_pandoc 'USE_PANDOC'
+define_in_config_if_yes $_curl 'USE_CURL'
```

Comment:
Instead of USE_CURL I would define DOWNLOAD_MANUAL or something like that

Reply:
I wanted to keep it as a general CURL binary check so it can be used for other things and not just downloading the manual.

Reply:
I said that having the building manual locally in mind. In the current case it's OK like that